cider-velox/test/expression/VeloxToCiderExpr.cpp

```diff
@@ -168,6 +168,14 @@ std::shared_ptr<Analyzer::Expr> VeloxToCiderExprConverter::toCiderExpr(
 std::shared_ptr<Analyzer::Expr> VeloxToCiderExprConverter::toCiderExpr(
     std::shared_ptr<const CallTypedExpr> vExpr,
     std::unordered_map<std::string, int> colInfo) const {
+  static const std::unordered_map<std::string, std::string> scalarMappings{
+      {"plus", "add"},
+      {"minus", "subtract"},
+      {"mod", "modulus"},
+      {"eq", "equal"},
+      {"neq", "not_equal"},
+      {"substr", "substring"},
+  };
```

Comment:
maybe this map should define inside FunctionLookupEngine?

Reply:
Done.

Reply:
should this map be removed here?